.codespell.exclude

```diff
@@ -0,0 +1 @@
+* Use American English for spelling and punctuation (ex. "color" instead of "colour"). This also means that text such as nicknames should be in double quotes (First "The Player" Last instead of First 'The Player' Last).
```

Comment:
```suggestion
* Use American English for spelling and punctuation (ex. `color` instead of `colour`). This also means that text such as nicknames should be in double quotes (First "The Player" Last instead of First 'The Player' Last).
```